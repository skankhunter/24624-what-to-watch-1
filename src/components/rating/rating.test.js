import React from "react";
import renderer from "react-test-renderer";

import Rating from "./rating.jsx";

const mocks = {
  activeStar: `1`,
  onStarClick: jest.fn()
};

describe(`Rating:`, () => {
  it(`Correctly renders after relaunch`, () => {
    const tree = renderer.create(<Rating {...mocks} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
