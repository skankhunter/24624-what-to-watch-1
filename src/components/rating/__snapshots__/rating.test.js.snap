// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating: Correctly renders after relaunch 1`] = `
<div
  className="rating"
>
  <div
    className="rating__stars"
  >
    <input
      checked={true}
      className="rating__input"
      id="star-1"
      name="rating"
      onChange={[Function]}
      type="radio"
      value={1}
    />
    <label
      className="rating__label"
      htmlFor="star-1"
    >
      Rating 
      1
    </label>
    <input
      checked={false}
      className="rating__input"
      id="star-2"
      name="rating"
      onChange={[Function]}
      type="radio"
      value={2}
    />
    <label
      className="rating__label"
      htmlFor="star-2"
    >
      Rating 
      2
    </label>
    <input
      checked={false}
      className="rating__input"
      id="star-3"
      name="rating"
      onChange={[Function]}
      type="radio"
      value={3}
    />
    <label
      className="rating__label"
      htmlFor="star-3"
    >
      Rating 
      3
    </label>
    <input
      checked={false}
      className="rating__input"
      id="star-4"
      name="rating"
      onChange={[Function]}
      type="radio"
      value={4}
    />
    <label
      className="rating__label"
      htmlFor="star-4"
    >
      Rating 
      4
    </label>
    <input
      checked={false}
      className="rating__input"
      id="star-5"
      name="rating"
      onChange={[Function]}
      type="radio"
      value={5}
    />
    <label
      className="rating__label"
      htmlFor="star-5"
    >
      Rating 
      5
    </label>
  </div>
</div>
`;
